.github/workflows/codestyle.yml

```diff
@@ -25,6 +33,9 @@ jobs:
       - name: Run Detekt
         run: |
             ./gradlew detektAll
+      - name: Run Linter
+        run: |
+            ./gradlew lint
```

Comment:
**Suggestion**: I think, it makes sense to create a job out of this, instead of sequential steps, so might be able to run these 2 in parallel.

Reply:
It does have the drawback that it will setup java and download dependencies twice in parallel, so the benefit is smaller.

But we do get lint / codestyle results faster.